src/Security/PasswordExpirationMiddleware.php

```diff
@@ -145,15 +145,15 @@ protected function handleExpiredPassword(HTTPRequest $request): ?HTTPResponse
             return null;
         }
 
-        $currentUrl = $this->absoluteUrl($request->getURL(true));
+        $currentUrl = $this->absoluteUrl((string) $request->getURL(true));
```

Comment:
```suggestion
        $currentUrl = $this->absoluteUrl($request->getURL(true));
```
Not necessary here

Reply:
Updated

Reply:
Actually adding it back it - it caused a unit test to fail - https://github.com/silverstripe/silverstripe-framework/actions/runs/3689681730/jobs/6245906511#step:12:116

Reply:
I'd be keen to understand how this cast-to-string is affecting that unit test - I would have expected the CI to fail by throwing an exception if the problem was that the wrong type is being passed to a strongly-typed parameter.